### DISCUSSION TITLE
Add Support For Projects Pages

### DISCUSSION THREAD

**Discussion by shimonelbaz** [EmilyRosina/gsf-docs#8] [Feedback: Idea]
the projects pages has the same filtering syntax but is not supported for the extensions

**Comment by EmilyRosina** [comment]
:wave: Hello, thanks for commenting!

However, I won't be adding support for the projects pages, my extension was only ever for issues and PRs list pages only.

Projects are great, but they have their own system entirely for filters and views etc, _far_ more powerful already than issues and PRs list pages. So I personally do not see the same benefits/gains... especially when also factoring in dev time and yet another page I will have to keep supporting and functioning well.
